03_Your_first_AI_training_job_on_LUMI/reference_solution/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+#SBATCH --account=project_465001063
+#SBATCH --partition=small-g
+#SBATCH --gpus-per-node=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=7
+#SBATCH --mem-per-gpu=60G
+#SBATCH --time=0:15:00
+
```

Comment:
Should this include a `--reservation` specification?

Reply:
done